src/mgr/DaemonServer.cc

```diff
@@ -2627,6 +2627,7 @@ void DaemonServer::send_report()
           tie(acc, std::ignore) = accumulated.emplace(metric.get_type(),
               std::move(collector));
         }
+        dout(4) << key << " " << metric << dendl;
```

Comment:
Is there a reason for `4`. Similar debugs around are on `10` (info). Also, might we worth to print `__func__`.